.github/workflows/python-app.yml

```diff
@@ -0,0 +1,33 @@
+name: Pycee2
```

Comment:
It would be interesting if the name of the action was a reflection of what it does. In this case, do you think it is reasonable to call this action something like ``Test and lint pycee2``?

Reply:
Done ;)